annotation_pipeline/record_types.txt

```diff
@@ -0,0 +1,36 @@
+Accident Reports
```

Comment:
any way we could keep this up to date programmatically, maybe checking when this is run? This adds another point of maintenance every time we edit the canonical `record_types` list. If there's not an easy fix, don't worry about it.